tests/core/test_actions.py

```diff
@@ -344,6 +362,227 @@ async def test_remote_action_with_template_param(
     ]
 
 
+@pytest.mark.parametrize(
+    "event",
+    (
+        EntitiesAdded(entities=[], timestamp=None),
+        DefinePrevUserUtteredFeaturization(
+            use_text_for_featurization=False, timestamp=None, metadata=None
+        ),
+        ReminderCancelled(timestamp=1621590172.3872123),
+        ReminderScheduled(
+            timestamp=None, trigger_date_time=datetime.now(), intent="greet"
+        ),
+        ActionExecutionRejected(action_name="my_action"),
+        LegacyFormValidation(validate=True, timestamp=None),
+        LoopInterrupted(timestamp=None, is_interrupted=False),
+        ActiveLoop(name="loop"),
+        LegacyForm(name="my_form"),
+        AllSlotsReset(),
+        SlotSet(key="my_slot", value={}),
+        SlotSet(key="my slot", value=[]),
+        SlotSet(key="test", value=1),
+        SlotSet(key="test", value="text"),
+        ConversationResumed(),
+        ConversationPaused(),
+        FollowupAction(name="test"),
+        StoryExported(),
+        Restarted(),
+        ActionReverted(),
+        UserUtteranceReverted(),
+        BotUttered(text="Test bot utterance"),
+        UserUttered(
+            text="hello",
+            parse_data={
+                "intent": {
+                    "id": -4389344335148575888,
+                    "name": "greet",
+                    "confidence": 0.9604260921478271,
+                },
+                "entities": [],
```

Comment:
can we please also have something with entities? They are super important in Rasa Open Source and should in my opinion be covered